app/helpers/member_helper.rb

```diff
@@ -76,6 +76,7 @@ def member_download_menu(member, parent:, labelled_by: nil, filename_base: nil,
       download_options.each do |option_config|
         list_elements << dropdown_menuitem(
           link_to(option_config[:url],
+            download: '', #forces the download of audio files
```

Comment:
Remove this -- we can modify the URL to force a download -- we already have code for this.